GliaWidgets/Sources/Component/OperatorTypingIndicator/OperatorTypingIndicatorView.swift

```diff
@@ -1,73 +1,159 @@
 import UIKit
-import Lottie
+
+private let animationDuration: CFTimeInterval = 0.5
 
 final class OperatorTypingIndicatorView: BaseView {
-    override var isHidden: Bool {
-        didSet {
-            isHidden ? animationView.stop() : animationView.play()
-        }
-    }
-    private let style: OperatorTypingIndicatorStyle
-    private let animationView = AnimationView()
-    private let kViewSize = CGSize(width: 28, height: 28)
-    private let kLeftInset: CGFloat = 10
-    private let bundleManaging: BundleManaging
+    lazy var dotLeadingView = DotView(
```

Comment:
Could we make dots `private`?

Reply:
fixed - a442c4ff7e7aeb045efb220807aee5e31d8fd6b9